geth/node/status_node.go

```diff
@@ -268,44 +263,26 @@ func (n *StatusNode) Config() (*params.NodeConfig, error) {
 	return n.config, nil
 }
 
-// LightEthereumService exposes reference to LES service running on top of the node
-func (n *StatusNode) LightEthereumService() (*les.LightEthereum, error) {
-	n.mu.RLock()
-	defer n.mu.RUnlock()
-
+func (n *StatusNode) service(s interface{}, name string) error {
```

Comment:
Even if it's also `Service()` at the geth node I would like a more descriptive name here or at least a better name for `s` and a useful comment.

Reply:
@themue what about this one?
```
// gethService is a wrapper for gethNode.Service wich retrieves a currently
// running service registered of a specific type.
func (n *StatusNode) gethService(serviceInstance interface{}, serviceName string) error {
... 
```

Reply:
Top, helps a lot.